cypress/integration/dashboard.spec.js

```diff
@@ -0,0 +1,18 @@
+describe("Dashboard", () => {
+    beforeEach(() => {
+        cy.setCookie("Other", "other");
+        cy.setCookie("XSRF-TOKEN", "abcde");
+        cy.visit("/", { failOnStatusCode: false });
```

Comment:
you can remove `{ failOnStatusCode: false }`, I had left it in when I was returning a 404 but ended up created a temp.gotmpl because of other reasons